README.md

```diff
@@ -1,18 +1,17 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# New York Times Article
 
-> One paragraph statement about the project.
+>  Clone a New York Times Article
 
 ![screenshot](./app_screenshot.png)
```

Comment:
- [ ] Add a screenshot of the page. (Replace the dummy screenshot currently present with a real one on the page)